README.md

```diff
@@ -139,12 +145,31 @@ for item in best:
 ```
 
 
-
 For detailed workflow check `Example.ipynb`.
 
+## Mating strategy
+The following mating strategies are available:
+* Union mating: If either (or both) of the parents have the feature selected the descendent will have it too.
+```python
+mating_strategy=UnionMating()
+```
+* Intersection mating: If both of the parents have the feature the descendent will have it too.
+```python
+mating_strategy=IntersectionMating()
+```
+* Intersection mating with information gain: If both of the parents have the feature the descendent will have it too. Additionally, some features from either one of the parents, that have the highest information gain are added.  
+```python
+mating_strategy=IntersectionMatingWithInformationGain()
+```
+* Intersection mating with weighted random information gain: If both of the parents have the feature the descendent will have it too. Additionally, some features from either one of the parents, will be added. The probbability of selecting a features is proportionate to this feature's information gain.
```

Comment:
This line might be misleading (it is similarly misleading in the docstring). The probability of a feature being selected is proportional to scaling function applied to it's calculated information gain. One probability has too many b-s.